app/views/spree/users/_transactions.html.haml

```diff
@@ -1,4 +1,28 @@
 %script{ type: "text/ng-template", id: "account/transactions.html" }
+  - if @payments_requiring_action.present?
+    .active_table
+      %h3.requiring-authorization= t(".authorization_required")
+      .small-12.columns
+        %table
+          %tr
+            %th.order1= t :transaction
+            %th.order2= t :transaction_date
+            %th.order3.show-for-large-up= t :payment_state
+            %th.order5.text-right= t :value
+          %tbody
+            - @payments_requiring_action.each do |payment|
+              %tr
+                %td.order1
+                  = link_to payment.order.number, main_app.order_path(payment.order)
+                %td.order2
+                  = payment.updated_at.strftime("%Y-%m-%d")
+                %td.order3
+                  %a{ href: "#{payment.cvv_response_message}" }
+                    %button.bright
+                      Authorize
+                %td.order5
+                  = payment.display_amount
+
```

Comment:
This doesn't use Angular, like the other table, but since we're working on removing it anyway I thought I'd make it more rails-y.